developer_manual/app/code_signing.rst

```diff
@@ -130,7 +130,7 @@ examples will assume that you are trying to sign an application named
    --certificate=/Users/lukasreschke/CA/contacts.crt --path=/Users/lukasreschke/Programming/contacts``
 
 The occ tool will store a ``signature.json`` file within the ``appinfo`` folder 
-of your application. Then compress the application folder and upload it to 
+of your application. Then compress the application folder (``contacts.tar.gz``) and upload it to 
```

Comment:
Ok. Then I'd change the wording in that sentence to "Then compress the application folder, naming it ``contacts.tar.gz``, and upload it to".